use-package.el

```diff
@@ -1406,6 +1408,67 @@ deferred until the prefix key sequence is pressed."
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
+;;; :customize
+;;
+
+(defun use-package-normalize/:customize (name-symbol keyword arg)
+  "Normalize use-package customize keyword."
+  (let ((error-msg (format  "%s wants a (<symbol> <form> <optional string comment>) or list of these" name-symbol)))
+    (unless (listp arg)
+      (use-package-error error-msg))
+    (dolist (def arg arg)
+      (unless (listp def)
+        (use-package-error error-msg))
+      (seq-let [variable value comment] def
+        (when (or (not variable)
+                  (not value)
+                  (> (length def) 3)
+                  (and comment (not (stringp comment))))
+          (use-package-error error-msg))))))
+
+(defun use-package-handler/:customize (name keyword args rest state)
+  "Generate use-package customize keyword code."
+  (let ((body (use-package-process-keywords name rest state)))
+    (use-package-concat
+     (seq-map (lambda (def)
```

Comment:
Assuming `seq.el` is available, is there a reason to prefer `seq-map` over `mapcar` here?

Reply:
When I finally have a nice sequence elisp library, I cannot use it anymore ;) I'm just using the seq-* function in my elisp because they seem more modern then the old disparate functions, but I understand the point here and will switch to mapcar.